app/config.py

```diff
@@ -25,7 +25,12 @@
     DB_CONFIG = (POSTGRES['user'], POSTGRES['host'], POSTGRES['database'])
     DATABASE_URL = "postgresql+psycopg2://{}@{}/{}".format(*DB_CONFIG)
 
+ISOTOPE_STORAGE_DIR = CONFIG['isotope_storage']['directory']
+ISOTOPE_S3_BUCKET = CONFIG['isotope_storage'].get('s3bucket')
```

Comment:
But what to do with the local VBox installation?

Reply:
This can be set to `None`, then it will not download anything.